src/main/java/roomescape/service/ReservationService.java

```diff
@@ -0,0 +1,77 @@
+package roomescape.service;
+
+import org.springframework.stereotype.Service;
+import roomescape.domain.Reservation;
+import roomescape.domain.ReservationTime;
+import roomescape.domain.Theme;
+import roomescape.dto.reservation.ReservationsResponse;
+import roomescape.dto.reservation.create.ReservationCreateRequest;
+import roomescape.dto.reservation.create.ReservationCreateResponse;
+import roomescape.dto.theme.ThemeResponse;
+import roomescape.dto.time.ReservationTimeResponse;
+import roomescape.exception.ErrorCode;
+import roomescape.exception.ErrorCodeResponse;
+import roomescape.exception.custom.InvalidReservationTimeException;
+import roomescape.exception.custom.ThemeNotFoundException;
+import roomescape.repository.ReservationRepository;
+import roomescape.repository.ReservationTimeRepository;
+import roomescape.repository.ThemeRepository;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class ReservationService {
+
+    private final ReservationRepository reservationRepository;
+
+    private final ReservationTimeRepository reservationTimeRepository;
+
+    private final ThemeRepository themeRepository;
+
+
+    public ReservationService(ReservationRepository reservationRepository, ReservationTimeRepository reservationTimeRepository,
+                              ThemeRepository themeRepository) {
+        this.reservationRepository = reservationRepository;
+        this.reservationTimeRepository = reservationTimeRepository;
+        this.themeRepository = themeRepository;
+    }
+
+    public List<ReservationsResponse> findReservations() {
+        return reservationRepository.findReservations()
+                .stream()
+                .map(ReservationsResponse::new)
+                .collect(Collectors.toList());
```

Comment:
> "Stream.toList()" method should be used instead of "collectors" when unmodifiable list needed

위와 같은 소나린트 경고문구가 뜨네요! 수정 불가능한 list 가 반환되도록 아래와 같이 수정되면 좋을 것 같아요 :) 
```suggestion
                .toList();
```